addons/sms/tests/test_sms_composer.py

```diff
@@ -0,0 +1,29 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.sms.tests.common import SMSCommon, SMSCase
+from odoo.tools import html2plaintext, plaintext2html
+
+
+class TestSMSComposerComment(SMSCommon, SMSCase):
+    """ Test behaviors that are overridden when other modules
+    are installed (e.g., mass_mailing). In these cases,
+    test_mail_sms or test_mail_full should be used."""
+
+    def test_body_has_no_link(self):
+        body = "Hello me, check this awesome app I found: https://odoo.com"
+        notification_body = plaintext2html(html2plaintext(body))
+        self.assertNotEqual(body, notification_body)
```

Comment:
only reading the test I don't know what's the actual content of `notification_body`, I suggest we change the assertion to `self.assertEqual(notification_body, ...)` (replace the `...`), one will figure out by himself that `body` and `notification_body` are different by reading the literal strings